extensions/wikia/Chat2/ChatBanListSpecial_ajax.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * @package MediaWiki
+ * @subpackage SpecialPage
+ * @author Piotr Molski <moli@wikia.com>
+ * @version: $Id$
+ */
+
+class ChatBanListAjax {
+
+	/**
+	 * ajax response for request params
+	 *
+	 */
+	public static function axShowUsers ( ) {
+		global $wgRequest, $wgUser, $wgCityId;
+
+		wfProfileIn( __METHOD__ );
+
+		$username 	= $wgRequest->getVal('username');
+		$limit		= $wgRequest->getVal('limit');
+		$offset		= $wgRequest->getVal('offset');
+		$loop		= $wgRequest->getVal('loop');
+		$order		= $wgRequest->getVal('order');
+
+        $result = [
+            'sEcho'                => intval($loop),
+            'iTotalRecords'        => 0,
+            'iTotalDisplayRecords' => 0,
+            'sColumns'             => '',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "sColumns" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)